draft-irtf-nwcrg-coding-and-congestion-04.txt

```diff
@@ -327,25 +369,32 @@ Internet-Draft            Coding and congestion                July 2020
 3.2.  Discussion
 
    While CC is in principle independent of other transport functions
-   such as reliability, we note that CC is often embedded in reliable
-   transfer protocols (e.g.  TCP).  This approach may require that the
-   transport protocol does not implement a fully reliable data transfer
+   such as connection establishment or reliability, we note that CC can
+   be coupled with a reliable mechanism.  This approach may require that
```

Comment:
I think that the first sentence here can just be removed now. It doesn't add value to the discussion in my opinion.
Also: why "may require", in the second sentence, and not "requires"? The second sentence talks about a "fully reliable transfer service" - adding FEC on top of a fully reliable channel is useless, period.  (right?)

Reply:
You are right - this has been reworked in the commit in #29